apidocs/getBalances.md

```diff
@@ -0,0 +1,117 @@
+
+---
+### [getBalances](https://github.com/iotaledger/iri/blob/dev/src/main/java/com/iota/iri/service/API.java#L989)
+ [AbstractResponse](/javadoc/com/iota/iri/service/dto/abstractresponse/) getBalancesStatement(java.util.List addrss, java.util.List tips, int threshold)
+
+Returns the confirmed balance, as viewed by tips, in case tips is not supplied, the balance is based on the latest confirmed milestone.
```

Comment:
Returns the confirmed balance, as viewed by tips, in case tips is not supplied, the balance is based on the latest confirmed milestone.
-> Returns the confirmed balance, as viewed by the specified `tips`. If you do not specify the referencing `tips`, the returned balance is based on the latest confirmed milestone.

...

 The balances is returned as a list in the same order as the addresses were provided as input.
->
 The balances **are** returned as a list in the same order as the addresses were provided as input.